android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -707,88 +707,93 @@ public static RNCWebViewModule getModule(ReactContext reactContext) {
-          else{
-            mWebView.setVisibility(View.GONE);
-          }
+            // Different root views, we are in a Modal
+            if(rootView.getRootView() != mWebView.getRootView()){
+              mWebView.getRootView().setVisibility(View.GONE);
+            }
 
-          mReactContext.addLifecycleEventListener(this);
-        }
+            // Same view hierarchy (no Modal), just hide the webview then
+            else{
+              mWebView.setVisibility(View.GONE);
+            }
 
-        @Override
-        public void onHideCustomView() {
-          if (mVideoView == null) {
-            return;
+            mReactContext.addLifecycleEventListener(this);
           }
 
-          // same logic as above
-          ViewGroup rootView = getRootView();
+          @Override
+          public void onHideCustomView() {
+            if (mVideoView == null) {
+              return;
+            }
 
-          if(rootView.getRootView() !=  mWebView.getRootView()){
-            mWebView.getRootView().setVisibility(View.VISIBLE);
-          }
+            // same logic as above
+            ViewGroup rootView = getRootView();
 
-          // Same view hierarchy (no Modal)
-          else{
-            mWebView.setVisibility(View.VISIBLE);
-          }
+            if(rootView.getRootView() !=  mWebView.getRootView()){
+              mWebView.getRootView().setVisibility(View.VISIBLE);
+            }
 
-          if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
-            mReactContext.getCurrentActivity().getWindow().clearFlags(WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS);
-          }
+            // Same view hierarchy (no Modal)
+            else{
+              mWebView.setVisibility(View.VISIBLE);
+            }
 
-          rootView.removeView(mVideoView);
-          mCustomViewCallback.onCustomViewHidden();
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
+              mReactContext.getCurrentActivity().getWindow().clearFlags(WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS);
+            }
 
-          mVideoView = null;
-          mCustomViewCallback = null;
+            rootView.removeView(mVideoView);
+            mCustomViewCallback.onCustomViewHidden();
 
-          mReactContext.getCurrentActivity().setRequestedOrientation(initialRequestedOrientation);
+            mVideoView = null;
+            mCustomViewCallback = null;
 
-          mReactContext.removeLifecycleEventListener(this);
-        }
-      };
-      webView.setWebChromeClient(mWebChromeClient);
+            mReactContext.getCurrentActivity().setRequestedOrientation(initialRequestedOrientation);
+
+            mReactContext.removeLifecycleEventListener(this);
+          }
+        };
+        webView.setWebChromeClient(mWebChromeClient);
+      }
+      else {
+        Log.w("RNCWebViewManager", "Unable to get current activity thus could not set web chrome client with full screen video.");
+      }
```

Comment:
This means the web chrome client won't be set at all, right? Is that a problem? Should there be some kind of fallback logic, e.g. to use the below branch as if fullscreen video wasn't enabled?

(I'm not sure, just asking questions here)

Reply:
Why do you think that play-services-tasks is related to react-native-webview and that this change would fix it?

Reply:
My bad, its a mistake, play console warning nothing to do with webview